### PR TITLE
Updates:

### PR DESCRIPTION
**Pull request by xfrankbx**
Added couple more Opcodes
Added Logger Critical, was missing, and fixed order to follow order defined above

### CODE REVIEW

File: Auth/Constants/LoginOpCode.py [hunk 1]
```diff
@@ -14,4 +14,6 @@ class LoginOpCode(Enum):
 
 class LoginResult(Enum):
     ''' Error codes '''
-    SUCCESS     = 0x00
+    SUCCESS             = 0x00
```
Comment (sergio-ivanuzzo) [inline Auth/Constants/LoginOpCode.py]: Currently there are no handlers for all cases except success. I think it's not necessary to add this now.

**Comment by sergio-ivanuzzo** [comment]
I have merged previous PR, it seems from here we can only get critical ?

File: Auth/Constants/LoginOpCode.py (hunk 1 above)
Comment (xfrankbx) [reply Auth/Constants/LoginOpCode.py]: Ah gotcha.  Will wait to add them with the handlers then.

**Comment by sergio-ivanuzzo** [comment]
Well, for now critical method added to the dev. Will be merged into master later. Thanks.

**Comment by xfrankbx** [comment]
Alrighty, sounds good to me :-)

**Pull request closed without merging**